Support/bone101/UI/bbui.js

```diff
@@ -108,8 +108,8 @@ var UI = (function() {
         // positions
         var BBposX = 283;
         var BBposY = 120;
-        var axisStartY = BBposY + 40;
-        var axisStartX = BBposX + 375;
+        var axisStartY = BBposY - 50;
```

Comment:
Why change these existing values? I think these came straight from the initial implementation.